packages/react/src/components/ErrorSummary/ErrorSummary.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Controls, Primary, ArgsTable, Canvas } from '@storybook/blocks';
+
+import * as ErrorSummaryStories from './ErrorSummary.stories';
+
+import { ErrorSummary } from './';
+
+<Meta of={ErrorSummaryStories} />
+
+# ErrorSummary
+
+<Primary />
+<Controls />
+
+## Bruk med tekstfelt
```

Comment:
Suggest adding an code snippet example and explanation under "Bruk" chapter, like we have for `List`.